frameworks/PHP/hyperf/benchmark_config.json

```diff
@@ -0,0 +1,30 @@
+{
+    "framework": "hyperf",
+    "tests": [
+        {
+            "default": {
+                "json_url": "/json",
+                "db_url": "/db",
+                "query_url": "/queries/",
+                "fortune_url": "/fortunes",
+                "update_url": "/updates/",
+                "plaintext_url": "/plaintext",
+                "port": 9501,
+                "approach": "Realistic",
+                "classification": "Fullstack",
+                "database": "MySQL",
+                "framework": "Hyperf",
+                "language": "PHP",
+                "flavor": "None",
+                "orm": "Full",
```

Comment:
It isn't a full ORM test.
In the _queries_ and _update_ tests, it is using `->to_array()`, so it isn't mapping the rows to objects.